index.js

```diff
@@ -199,6 +212,11 @@ module.exports = (opts = {}) => {
           'add-mixin': (node, helpers) => {
             insertMixin(helpers, mixins, node, opts)
           }
+        },
+        OnceExit(_, helpers) {
+          if (opts.mixinsDir && opts.mixinsDir.length > 0) {
+            watchNewMixins(helpers, opts.mixinsDir, opts.parent, MIXINS_GLOB)
```

Comment:
`parent` is CSS file, which was a reason why plugin added this dependency.

Do not set it for the `postcss-mixins` case since we set dependency because of the plugin option, not because of input CSS

Reply:
Did I understand correctly?

Reply:
Nope. Remove `parent` key from `dir-dependency`.